### DISCUSSION TITLE
found no layout file for "HTML" for kind "section": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.

### DISCUSSION THREAD

**Discussion by Huoke** [HEIGE-PCloud/DoIt#448] [Q&A]
Windows 10 platform， hugo + DoIt,  run hugo
WARN 2022/01/10 00:39:36 found no layout file for "HTML" for kind "section": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.

**Comment by Huoke** [comment]
![image](https://user-images.githubusercontent.com/24191613/148693386-a334ddf5-404c-4b75-97bb-2a9eed569b5b.png)

**Comment by HEIGE-PCloud** [comment]
Use `git clone --recursive` to clone your repo if you imported DoIt as a submodule.

If you have already cloned the repo, run `git submodule update --init --recursive` to init the submodules.

**Comment by Huoke** [comment]
config.toml file configure error cause this error，thanks

**Comment by mlazzarotto** [comment]
I have the same error, but running `git submodule update --init --recursive` from within _quickstart_ folder apparently did nothing
